requirements.txt

```diff
@@ -3,6 +3,7 @@ git+https://github.com/JelleAalbers/blueice
 h5py
 iminuit
 git+https://github.com/XENONnT/inference_interface
+iminuit >= 2.21
```

Comment:
Usually, we only need `iminuit>=2.21`. Reference: https://pip.pypa.io/en/stable/user_guide/#requirements-files

I also prefer `iminuit>=2.21.0` to indicate that `iminuit` has a patch version.

Reply:
You mean instead of writing `iminuit >= 2.21` write `iminuit>=2.21.0`?

Reply:
Yes.

Reply:
ok fine